### PR TITLE
Track and log number of sessions on creation/deletion

### PR DESCRIPTION
**Pull request by jerryblakley**

### CODE REVIEW

**Comment by jerryblakley** [comment]
@kibra review?

File: src/session.c [hunk 1]
```diff
@@ -336,13 +346,17 @@ static CURL *session_get_temp_handle(void) {
 
     log_print(LOG_INFO, SECTION_SESSION_DEFAULT, "Opening temporary cURL session.");
     session = curl_easy_init();
+    update_session_count(true);
```
Comment (kibra) [inline src/session.c]: What if session == NULL? I don't think we'll ever end up decrementing `session_count`.

File: src/session.c [hunk 2]
```diff
@@ -325,6 +334,7 @@ static CURL *session_get_handle(bool new_handle) {
     log_print(LOG_INFO, SECTION_ENHANCED, "Opening cURL session -- fusedav.%s.sessions:1|c", filesystem_cluster);
     log_print(LOG_INFO, SECTION_SESSION_DEFAULT, "Opening cURL session");
     session = curl_easy_init();
+    update_session_count(true);
```
Comment (kibra) [inline src/session.c]: What if session == NULL? I don't think we'll ever end up decrementing `session_count`.

**Comment by kibra** [comment]
+1

:honey_pot:

**Pull request merged**